pkg/apis/kubeone/types.go

```diff
@@ -255,7 +255,36 @@ type ClusterNetworkConfig struct {
 	// CNI
 	// default value is {canal: {mtu: 1450}}
 	CNI *CNI `json:"cni,omitempty"`
-}
+	// KubeProxy
+	KubeProxy *KubeProxyConfig `json:"kubeProxy,omitempty"`
+}
+
+type KubeProxyConfig struct {
+	IPVS     *IPVSConfig `json:"ipvs"`
+	IPTables *IPTables   `json:"iptables"`
+}
+
+type IPVSConfig struct {
+	// ipvs scheduler
+	Scheduler string `json:"scheduler"`
+	// excludeCIDRs is a list of CIDR's which the ipvs proxier should not touch
+	// when cleaning up ipvs services.
+	ExcludeCIDRs []string `json:"excludeCIDRs"`
```

Comment:
Similar to the comment above, we could add some validation for this field as well.

Reply:
Nether this one have an upstream validations.